pages/05.governance/01.project-governance/governance.md

```diff
@@ -415,7 +415,166 @@ A person who would bring the Community Council above the limit on co-affiliation
 
 For purposes of this Section, a common affiliation includes all organizations that, directly or indirectly through one or more intermediary controls, is controlled by, or is under common control with the other entities declared as affiliations by other members of the Community Council.
 
-## 6. Record keeping
+## 6. Contributors, Maintainers and the Core Team
+
+### 6.1 Contributors
+Contributors are people who contribute their work to Mautic. This includes but is not limited to:
+
+- Code contributions, 
+- Writing documentation,
+- Submitting bug reports,
+- Other issue reports,
+- Reviewing PRs,
+- Participation in technical as well as non-technical discussions, 
+- Organizational considerations.
+
+Code contributions are very welcome, they are the life-blood of our open source project. In order to streamline and harmonize code quality, contributors must follow the contributing guidelines.
+
+Contributors may be associated with organizations - by employment or otherwise - who have a vested interest in Mautic, or may be individuals who have their own personal stakes in Mautic. We call these organizations and individuals 'stakeholders' throughout this section of the governance model to summarize them.
+
+### 6.1.1 Expectations of contributors
+- Be empathetic and respectful to the reviewers. Reviewing a change can be hard work and time-consuming.
+- Use the PR template in its entirety and provide very clear, step by step instructions on how to reproduce the bug you're fixing  if it's a bugfix, and how your contribution should work/what the feature is you're adding. Screenshots and screen recordings help enormously! 
+- Don't assume the reviewer is a developer, they may be a marketer helping with a user review.
+- Keep commits small when possible and provide reasoning and context when submitting changes. Reviews go smoother if you make the reviewer's job easier.
+- Be responsive when changes are requested by the reviewer. It is easier to re-review the modified changes if they are completed shortly after original review.
+- Ask for clarification if you are confused by a suggested change.
+- Speak up if your contribution appears to be stuck for more than a week. Post it in #t(Opens in new tab)-product on Slack(External link) and ask for assistance to move it forward.
```

Comment:
Just commenting here for exposure and reminder. There are no links here yet.

Reply:
Whoops, this was a copy/paste from the debate! Good catch!